GameServerLib/Logic/Chatbox/Commands/HelpCommand.cs

```diff
@@ -4,37 +4,40 @@
 
 namespace LeagueSandbox.GameServer.Logic.Chatbox.Commands
 {
-    public class HelpCommand : ChatCommand
+    public class HelpCommand : ChatCommandBase
     {
-        public HelpCommand(string command, string syntax, ChatCommandManager owner) : base(command, syntax, owner) { }
+        private readonly Game _game;
+
+        public override string Command => "help";
+        public override string Syntax => "";
+
+        public HelpCommand(ChatCommandManager chatCommandManager, Game game) : base(chatCommandManager)
+        {
+            _game = game;
+        }
 
         public override void Execute(Peer peer, bool hasReceivedArguments, string arguments = "")
         {
-            var _game = Program.ResolveDependency<Game>();
             if (!_game.Config.ChatCheatsEnabled)
             {
-                _owner.SendDebugMsgFormatted(DebugMsgType.INFO, "Chat commands are disabled in this game.");
+                ChatCommandManager.SendDebugMsgFormatted(DebugMsgType.INFO, "Chat commands are disabled in this game.");
                 return;
             }
 
-            var _chatboxManager = Program.ResolveDependency<ChatCommandManager>();
-
             var commands = "";
             var count = 0;
-            foreach (var command in _owner.GetCommandsStrings())
+            foreach (var command in ChatCommandManager.GetCommandsStrings())
             {
                 count++;
                 commands = commands
```

Comment:
```CS
commands = string.Format(
    "{0}<font color =\"#E175FF\"><b>{1}{2}</b><font color =\"#FFB145\">, "
    commands,
    ChatCommandManager.CommandStarterCharacter
    command
);
```
For example

Another good option would be to use the $ syntax

The real solution would be to have helper functions/enums for creating colors so you could use `FontColor.Orange` instead of `<font color =\"#FFB145\">` or something along the lines.

The way I'd write this would be
```CS
public override void Execute(Peer peer, bool hasReceivedArguments, string arguments = "")
{
    if (!_game.Config.ChatCheatsEnabled)
    {
        ChatCommandManager.SendDebugMsgFormatted(DebugMsgType.INFO, "Chat commands are disabled in this game.");
        return;
    }

    var commands = ChatCommandManager.GetCommandsStrings();
    var commandPrefix = "<font color =\"#E175FF\"><b>"
    var commandSuffix = "</b><font color =\"#FFB145\">"
    var commandList = commandPrefix + string.Join($"{commandPrefix}, {commandSuffix}", commands) + commandSuffix

    ChatCommandManager.SendDebugMsgFormatted(DebugMsgType.INFO, "List of available commands: ");
    ChatCommandManager.SendDebugMsgFormatted(DebugMsgType.INFO, commandList);
    ChatCommandManager.SendDebugMsgFormatted(DebugMsgType.INFO, $"There are {commands.Count} commands");
}
```